lib/redmine/scm/adapters/subversion_adapter.rb

```diff
@@ -84,10 +84,10 @@ def info
               # root_url = doc.elements["info/entry/repository/root"].text
               info = Info.new(root_url: doc['info']['entry']['repository']['root']['__content__'],
                               lastrev: Revision.new(
-                                 identifier: doc['info']['entry']['commit']['revision'],
-                                 time: Time.parse(doc['info']['entry']['commit']['date']['__content__']).localtime,
-                                 author: (doc['info']['entry']['commit']['author'] ? doc['info']['entry']['commit']['author']['__content__'] : '')
-                               )
+                                identifier: doc['info']['entry']['commit']['revision'],
+                                time: Time.parse(doc['info']['entry']['commit']['date']['__content__']).localtime,
```

Comment:
Line is too long. [114/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.